### PR TITLE
🗺 Station Map

### LINKED ISSUE #2714: Update AAPB map (American Samoa, Puerto Rico) (2)
### Because

_No response_

### Done when

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mrharpo**
# Station Map
New station map, now with actual map!

```[tasklist]
### Features
- [x] OpenStreetMap
- [x] State boundaries
- [x] Click action per region
- [x] Pins for stations 
- [x] Resolve current stations
- [x] Integrate into `Home` page
- [x] Integrate into `/participating-orgs` page 
- [x] New Stations
- [x] American Samoa
- [x] Puerto Rico
- [x] Guam
```
```[tasklist]
### TODO
- [x] Serve json datasets as static assets from webserver
- [x] On State click: ~go to /participating-orgs#State~ Display popup with station list for that state
- [x] Figure out how to get Guam to show up on the left Pacific, not just the right Pacific (Because it's on the other side of the International Date Line)
```
```[tasklist]
### Style
- [x] Participating Orgs section on Home page disappears entirely on mobile (< 768px)
- [x] Fix geo grouping squares to circles
- [ ] Better colors for geo clusters, by size
- [x] Popup box:
- [x] Too big. Opens beyond screen.
- [x] Consistent logo sizing
- [x] Large descriptions should scroll / be truncated
- [x] Better indicate external link to station
- [x] Add link to search for station in catalog
- [ ] Make map wider on page
- [x] Geo-Lock view 
- [ ] Change station markers to something cooler?
```
```[tasklist]
### Data
- [ ] Verify this station list against the solr index for particiapting orgs 
- [x] Add new stations to map
- [x] Update station count on home page: `More than 120` -> `More than 200`
- [x] Add GBH to list
```

Closes #2714

### CODE REVIEW

**Comment by ekemeyer** [comment]
Feedback from April 30 meeting with MLA/Library of Congress:
It looks great! Huge improvement from what we currently have!

Individual station pop ups:

- [ ] Change pop ups to only include station logo, link to main AAPB "splash" page, location. 
- [ ] Remove any instances of descriptions and/or links taking you directly to station website

State-wide pop ups:

- [ ] Clicking on state should mirror current prod behavior - user should be directed to page with alphabetical lists of stations organized by state, with URL links taking you to AAPB splash page for the station
- [ ] Remove the popup containing the list and links for all stations in that state

**Comment by ekemeyer** [comment]
@mrharpo, final request:

- [ ] start with wider view by default so as to include alaska, hawaii and other territories

File: cmless.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from os import path
+
+
+def parse_cmless(md):
+    md = md.replace('’', "'")  # Replace curly apostrophes with straight ones
+    md = md.replace('\\_', '_')  # Replace escaped underscores with plain ones
+    md = re.sub(
+        r"(?<!\n)\n(?!\\n)", " ", md
+    )  # Replace newlines that aren't escaped with spaces
+    pattern = r"## (.*?)\n(.*?)(?=## |\Z)"
```
Comment (afred) [inline cmless.py:11]: just checking my regex grokking... does this grab sections by the `##` header?

Comment (mrharpo) [reply cmless.py:11]: Yep! I forget the specifics of how it works, but it was able to successfully parse chunks of cmless and turn them into `key: value` pairs for all the markdown files.

Code was provided courtesy of (and thanks to) Copilot.

**Pull request merged**